ptychonn/position/reconstructor.py

```diff
@@ -0,0 +1,44 @@
+import torch
+import numpy as np
+
+from ptychonn.position.configs import InferenceConfig
+from ptychonn.position.io import *
+
+
+class Reconstructor:
+    def __init__(self, config_dict: InferenceConfig):
+        """
+        Inference engine for PtychoNN.
+
+        :param config_dict: dict. Configuration dictionary.
+        """
+        self.config_dict = config_dict
+        self.device = None
+
+    def build(self):
+        if torch.cuda.is_available():
+            self.device = torch.device("cuda")
+        else:
+            self.device = torch.device("cpu")
+
+    def batch_infer(self, x):
+        pass
```

Comment:
Can unused definitions like this one be removed?

Reply:
Removed unused definitions and modules in bc80a68.